reconcile/test/test_terraform_users.py

```diff
@@ -55,3 +64,116 @@ def test_send_email_invites_skip(mocker, new_users):
     sm = mocker.patch("reconcile.terraform_users.SmtpClient", autospec=True)
     send_email_invites(new_users, sm, [])
     sm.send_mails.assert_not_called()
+
+
+@pytest.fixture
+def pgp_reencryption_settings(
+    gql_class_factory: Callable[..., PgpReencryptionSettingsQueryData],
+) -> PgpReencryptionSettingsQueryData:
+    return gql_class_factory(
+        PgpReencryptionSettingsQueryData,
+        {
+            "pgp_reencryption_settings": [],
+        },
+    )
+
+
+@pytest.fixture
+def test_aws_account_role() -> dict:
+    return {
+        "name": "test_aws_account",
+        "users": [{"name": "test-user"}],
+        "aws_groups": [
+            {
+                "name": "test-group",
+                "account": {
+                    "name": "test-account",
+                },
+            }
+        ],
+        "user_policies": [
+            {
+                "name": "test-policy",
+                "account": {
+                    "name": "test-account",
+                },
+            },
+        ],
+    }
+
+
+@pytest.fixture
+def test_aws_account() -> dict:
+    return {
+        "name": "test-account",
+    }
+
+
+def test_setup(
+    mocker: MockerFixture,
+    test_aws_account: dict,
+    test_aws_account_role: dict,
+) -> None:
+    mocked_gql_api = create_autospec(GqlApi)
```

Comment:
nit: Wondering if it would make sense to move [gql_api_builder](https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/test/test_typed_queries/conftest.py#L28) to the global conftest.py so it can be used everywhere?

Reply:
sure, I was thinking integration should use a typed query instead of calling it directly, but apparently it's not the current state.

Reply:
`gql_api_builder` moved to global https://github.com/app-sre/qontract-reconcile/pull/3640/commits/7c00dee48a4136a3c335a1fe9908e49fb90b6a48